src/main/java/namor/template/calculator/RPNcalculator.java

```diff
@@ -0,0 +1,65 @@
+package namor.template.calculator;
+
+import namor.template.stack.ArrayStack;
+
+public class RPNcalculator {
+    private ArrayStack numbers = new ArrayStack();
```

Comment:
The last change in this task is to avoid here global variable. Make code in such way that Stack creates and use only in methods.